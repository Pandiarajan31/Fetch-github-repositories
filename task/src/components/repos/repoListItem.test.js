import React from 'react';
import RepoListItem from './repoListItem';
import renderer from 'react-test-renderer'


//   renders correctly when change the viewdRepo count

it('renders correctly when change the viewdRepo count', () => {

    const component =renderer.create(
        <RepoListItem/>
    )
    const instance=component.getInstance()
    expect(instance.state.viewdRepo).toBe(0);
    instance.countViewdRepo()
    expect(instance.state.viewdRepo).toBe(1);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });