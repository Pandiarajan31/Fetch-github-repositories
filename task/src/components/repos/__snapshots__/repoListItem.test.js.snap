// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when change the viewdRepo count 1`] = `null`;
